### DISCUSSION TITLE
Can I retrieve files uploaded from a deleted app?

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#12083] [Help] [answered]
Hello,

I wrongly deleted an app where users could upload some documents in an internal budibase table with the Attachment table type. But now I want to get all the documents I uploaded in an internal table. I have the app ID and the table ID yet. Is there any solution to retrieve all the documents from that deleted table?

Thank you for the support

**Comment by melohagan** [comment, marked as answer]
Hey @luigiPinnaAevo 

Unfortunately for your case the whenever you delete an app or table, any associated attachments will also be deleted.
